src/ResourceManager/Security/Commands.Security/help/Get-AzureRmDiscoveredSecuritySolution.md

```diff
@@ -1,6 +1,6 @@
 ---
 external help file: Microsoft.Azure.Commands.SecurityCenter.dll-Help.xml
-Module Name: AzureRM.Security
+Module Name: Microsoft.Azure.Commands.SecurityCenter
```

Comment:
Az.Security - this actually needs to match the physical module name, not the module description or assembly name